LayoutTests/fast/text/text-combine-text-transform-expected.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<body style="-webkit-writing-mode: vertical-rl">
```

Comment:
```suggestion
<body style="writing-mode: vertical-rl">
```
Can we use the standard version of the properties?

Reply:
Will do once run through EWS and if there is any need to rebaseline any other tests. So I can give it a go in one attempt. :-)

Reply:
Fixed.